src/Menu/Menu.js

```diff
@@ -103,6 +103,11 @@ class Menu extends React.Component {
     initiallyKeyboardFocused: React.PropTypes.bool,
 
     /**
+    * If true, the menu will be focused by the keyboard.
+    */
+    keyboardFocused: React.PropTypes.bool,
```

Comment:
Could you add a default value inside the `static defaultProps = {`?
That's for the documentation.